docs/building-apps/network-features/vetkeys/introduction.mdx

```diff
@@ -10,89 +10,42 @@ import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
-Following standard practices in [identity-based encryption (IBE) schemes](/docs/references/vetkeys-overview), the derived keys are securely transported to users in encrypted form. This ensures that neither individual nodes nor the network ever have access to a user’s private keys.
+- **Encrypted**: Following standard practices in [identity-based encryption (IBE) schemes](/docs/references/vetkeys-overview), the derived vetKeys are securely transported to users in encrypted form. This ensures that neither individual nodes nor the network ever have access to a user’s private vetKeys.
 
-## vetKeys use cases and applications
-
-The availability of vetKeys allows for a series of applications, including but not limited to those covered in the following sections.
-
-### [Distributed key management service (DKMS)](/docs/building-apps/network-features/vetkeys/dkms/introduction)
-
-`DKMS` is a higher-level library built on top of `vetKeys`, designed to offer simple, developer-friendly APIs especially for those without cryptographic expertise. It allows users to securely generate, manage, and share cryptographic keys using `vetKeys` and ICP. These keys can then be used for encryption, decryption, or data signing.
-
-DKMS has two primary components:
-- **[`KeyManager`](dkms/key-manager.mdx)**: Access control and vetKey derivation.
-- **[`EncryptedMaps`](dkms/encrypted-maps.mdx)**: Built on top of the `KeyManager` for efficient data encryption and storage in a key-value store.
-
-One of the primary use cases of `DKMS` is to enable a blockchain to securely encrypt and host data at scale to support millions of users and billions of secrets. Because BLS signatures are unique, they can be directly used as symmetric keys. For example:
-
-- **Secure file storage**: In a secure file storage dapp, users can encrypt their files using `DKMS` before uploading them. The dapp ensures that only authorized users can decrypt the files. While blockchain nodes assist in key recovery, they never see the private keys or decrypted file contents.
-
-- **Secure messaging**: A secure messaging dapp can encrypt conversations using the identities of the two participants. The dapp manages access so only those users can retrieve the decryption keys.
-
-- **Decentralized social networking (SocialFi)**: In a decentralized social network, posts can be encrypted with a key tied to a unique post identifier. The SocialFi dapp ensures that only the author and selected recipients have access to the corresponding decryption key.
-
-### [Blockchain-issued BLS signatures](/docs/building-apps/network-features/vetkeys/advanced/bls-signatures)
+- **Threshold**: As vetKeys are computed in a distributed manner across ICP nodes, there is no central authority generating vetKeys for users.
```

Comment:
Not 100% happy, but I'd like to stress that a threshold of parties is necessary for the derivation and to mention that this happens in a subnet.  

```suggestion
- **Threshold**: vetKeys are derived in a distributed manner, requiring a contribution of a threshold of nodes in a subnet. No single ever sees the derived vetKeys, ensuring that no central authority can derive or reconstruct them.
```